demo-app/src/main/kotlin/io/getstream/video/android/ui/join/CallJoinScreen.kt

```diff
@@ -234,7 +234,7 @@ private fun CallJoinHeader(
         Text(
             modifier = Modifier.weight(1f),
             color = Color.White,
-            text = user?.name?.ifBlank { user?.id }?.ifBlank { user!!.custom["email"] }.orEmpty(),
+            text = user?.userNameOrId ?: "",
```

Comment:
We can make this `user?.userNameOrId.orEmpty()`

Reply:
Thanks for noticing. Used it in another place, but missed these ones. 👍